### PR TITLE
Adding unitary hack banner

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

File: jinja/index.html [hunk 1]
```diff
@@ -14,6 +14,32 @@
...
+
+        <div class="row pb-5 align-items-center justify-content-center">
+            <div class="m-3 card-image">
+                <a href="https://github.com/PennyLaneAI/pennylane/blob/master/unitaryhack.md">
+                    <img width="200px" src="https://unitaryfund.github.io/unitaryhack/assets/logo-date.png" />
+                </a>
+            </div>
+            <div class="m-3 text-center align-items-center">
+                <p class="black-text">
+                    <strong>PennyLane is taking part in unitaryHACK.</strong>
```
Comment (co9olguy) [inline jinja/index.html:27]: Could we have this part in a larger font size? Maybe 1.5-2x the current height

**Review by co9olguy — CHANGES_REQUESTED** [review]
Thanks @nickx720! Looking good. Just one request about increasing the main bolded part to a larger font size (looks a bit dimunitive at the moment)

File: jinja/index.html (hunk 1 above)
Comment (co9olguy) [reply jinja/index.html:27]: The same font as in the headers of the cards below ("Learn", "Play", "Hack") might do the trick

Comment (nickx720) [reply jinja/index.html:27]: Sure @co9olguy will do those changes.

**Comment by josh146** [comment]
Hey @nickx720, it might look slightly cleaner if we move the link underneath like so:
image

Other than that, looking really nice

File: jinja/index.html [hunk 2]
```diff
@@ -14,6 +14,34 @@
...
+            <div class="m-3 text-center align-items-center">
+                <h3 class="card-title"></i>
+                    <strong>PennyLane is taking part in unitaryHACK.</strong>
+                </h3>                
+                <p class="black-text">
+                    <br>Check out our list of eligible issues.
+                </p>
+
+            </div>
+            <div class="m-3">
```
Comment (co9olguy) [inline jinja/index.html:34]:
```suggestion
```

File: jinja/index.html [hunk 3]
```diff
@@ -14,6 +14,31 @@
...
+                <a href="https://github.com/PennyLaneAI/pennylane/blob/master/unitaryhack.md">
+                    <img width="200px" src="https://unitaryfund.github.io/unitaryhack/assets/logo-date.png" />
+                </a>
+            </div>
+            <div class="m-3 text-center align-items-center">
+                <h3 class="card-title"></i>
+                    <strong>PennyLane is taking part in unitaryHACK.</strong>
+                </h3>                
+                <p class="black-text">
+                    <br>Check out our list of eligible issues.
```
Comment (co9olguy) [inline jinja/index.html:30]:
```suggestion
                    <br>A quantum open-source hackathon, running from May 14-30. Win swag and claim bounties for contributing to open-source projects.
```

File: jinja/index.html [hunk 4]
```diff
@@ -14,6 +14,31 @@
...
+                <a href="https://github.com/PennyLaneAI/pennylane/blob/master/unitaryhack.md">
+                    <img width="200px" src="https://unitaryfund.github.io/unitaryhack/assets/logo-date.png" />
+                </a>
+            </div>
+            <div class="m-3 text-center align-items-center">
+                <h3 class="card-title"></i>
+                    <strong>PennyLane is taking part in unitaryHACK.</strong>
+                </h3>                
+                <p class="black-text">
+                    <br>A quantum open-source hackathon, running from May 14-30. Win swag and claim bounties for contributing to open-source projects.
```
Comment (co9olguy) [inline jinja/index.html:30]:
```suggestion
                    <br>A quantum open-source hackathon, running from May 14-30. 
                    <br>Win swag and claim bounties for contributing to open-source projects.
```

**Review by co9olguy — APPROVED** [review]

**Pull request merged**